tests/e2e/lib/container

```diff
@@ -123,6 +125,7 @@ create_qm_node() {
                 ContainerFile.node"${nodeID}" \
                 "node${nodeID}" \
                 "node:latest" \
+                "SYS_ADMIN /tmp/"
```

Comment:
How about we use a global variable to this patch?

Reply:
This parameter should be defined only at the creation of nodes not the controllers
So we parse it as parameter,

Can you explain with snippet?

Reply:
like this: https://github.com/containers/qm/blob/main/tests/e2e/run-test-e2e#L28

Reply:
ACK